nextcloudappstore/user/migrations/0001_initial.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 4.2.16 on 2024-12-06 11:59
+
+import django.db.models.deletion
+from django.conf import settings
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="UserProfile",
+            fields=[
+                ("id", models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name="ID")),
+                (
+                    "subscribe_to_news",
+                    models.BooleanField(default=True, help_text="User agrees to receive Nextcloud news and updates."),
```

Comment:
Similarly, `User has opted in to receive Nextcloud news and updates.` might be a better sentence here.